db/submitted_plugins.py

```diff
@@ -20,3 +20,24 @@ def insert(plugin_data):
         plugin_data['submitted_at'] = int(time.time())
 
     r.table('submitted_plugins').insert(plugin_data).run(r_conn())
+
+
+def get_list():
+    return list(r.table('submitted_plugins')
+                .order_by(index=r.desc('submitted_at'))
+                .filter(r.row['approved'] != True, default=True)  # NOQA
+                .run(r_conn()))
+
+
+def get_by_id(id):
+    return r.table('submitted_plugins').get(id).run(r_conn())
+
+
+def delete(id):
+    return r.table('submitted_plugins').get(id).delete().run(r_conn())
+
+
+def approved(id):
```

Comment:
can you call this `set_approved` or just `approve`

Reply:
I renamed it and introduced some additional logic here that will enable scraping.